doc/api-documentation.rst

```diff
@@ -53,4 +53,11 @@ API Documentation
 .. automodule:: oled.virtual
     :members:
     :undoc-members:
-    :show-inheritance:
\ No newline at end of file
+    :show-inheritance:
+
+:mod:`oled.error`
```

Comment:
yes will do. Perhaps we should also limit the API to the parts that developers actually use, so hide the mixins, threading etc module?